src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2341,6 +2481,28 @@
           }
         },
 
+        "requests": {
+          "description": "Requests associated with this run instance, if any.",
```

Comment:
>Requests [](start = 26, length = 8)

I like the formulation above, used in threadFlowLocations, better: "An array of <objectType> objects cached at run level"